docs/blog/posts/2023-05-24-latest_developments.md

```diff
@@ -0,0 +1,137 @@
+---
+date: 2022-07-14
+authors:
+  - ross-k
+  - robin-l
+categories:
+  - Feature Updates
+---
+
+# Splink Updates - July 2023
+
+** :new: Welcome to the Splink Blog! :new: **
+
+Its hard to keep up to date with all of the new features being added to Splink, so we have launched this blog to share a round up of latest developments every few months.
+
+So, without further ado, here are some of the highlights from the first half of 2023!
+
+<!-- more -->
+
+Latest Splink version: [v3.9.4](https://github.com/moj-analytical-services/splink/releases/tag/v3.9.4)
+
+
+## :rocket: Massive speed gains in EM training
+
+There’s now an option to make EM training much faster - in one example we’ve seen at 1000x fold speedup.  Kudos to external contributor [@aymonwuolanne](https://github.com/moj-analytical-services/splink/pull/1369) from the Australian Bureau of Statistics! 
+
+To make use of this, set the `estimate_without_term_frequencies` parameter to True, for example:
+
+```py
+linker.estimate_parameters_using_expectation_maximisation(..., estimate_without_term_frequencies=True)
```

Comment:
Do we have anywhere to link with caveat about possible parameter accuracy? Just that if reading this I might wonder about why I would have to opt-in to get such a big speedup

Reply:
Good point - do you think we should just include a caveat here? Or add a caveat in a more permanent home in the docs and link to it from here?

Reply:
It is mentioned in the docs of the [parameter in estimate_parameters_using_expectation_maximisation](https://moj-analytical-services.github.io/splink/linkerest.html#splink.linker.Linker.estimate_parameters_using_expectation_maximisation) so my first instinct is to add a caveat here and refer to these API docs.

Reply:
Yep, totally makes sense